Gemfile

```diff
@@ -8,6 +8,11 @@ install_if -> { RUBY_VERSION > '3.1' } do
   gem "net-smtp"
 end
 
+# switch to install_if when ruby 2.2 support is dropped
```

Comment:
Please do switch to install_if now, there's one just above this block now

Reply:
Done, and thanks for dropping 2.2 from the build.  I was surprised the bundler that came with ruby 2.2 couldn't handle the `install_if` version, but I didn't want to resort to multiple gemfiles!